src/App.tsx

```diff
@@ -1,25 +1,122 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api';
 
 const App: React.FC = () => {
   const [
     selectedUserId,
-    // setSelectedUserId,
+    setSelectedUserId,
   ] = useState(0);
 
+  const [
+    selectedTodoId,
+    setSelectedTodoId,
+  ] = useState(0);
```

Comment:
now your variables are not that long, let's write all setStates in one line

```suggestion
  const [selectedTodoId, setSelectedTodoId] = useState(0);
```

Reply:
Done